back/app/services/content_image_service.rb

```diff
@@ -37,6 +37,8 @@ def swap_data_images(imageable, field)
 
   def render_data_images(imageable, field)
     multiloc = imageable.send field
+
+    return multiloc if multiloc.nil?
```

Comment:
> I think we had one or two bugs in the past where the backend would crash on a multiloc that has a nil value.

I think that is exactly what happened here and why I added it- don't remember exactly why or in which situations it would happen though